artemis-server/src/main/java/org/apache/activemq/artemis/core/settings/impl/AddressSettings.java

```diff
@@ -1604,6 +1619,10 @@ public void decode(ActiveMQBuffer buffer, boolean tryCompatible) {
          prefetchPageMessages = BufferHelper.readNullableInteger(buffer);
       }
 
+      if (buffer.readableBytes() > 0) {
+         intermediateMessageBufferInitialSize = BufferHelper.readNullableInteger(buffer);
+      }
+
```

Comment:
Notice the comment below. This code should be removed.

Reply:
Missed that—done now.